tests/core/views/test_generic.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+from tests.conftest import client
+
+@pytest.mark.asyncio
+async def test_core_views_generic(client):
+    response = await client.get("/")
+    assert response.status_code == 200
+    assert "Add a new wallet" in response.text
```

Comment:
I think we should remove these string tests and just test for http code. Later, we we'll add more tests here.

Reply:
Sure, I'll remove the string assertions.